src/saving/SaveStatusOverlay.cs

```diff
@@ -86,13 +73,7 @@ public void ShowMessage(string message, float visibleTime = 0.7f)
     public void ShowError(string title, string message, string exception, bool returnToMenu = false,
         Action onClosed = null)
     {
-        errorDialog.WindowTitle = title;
-        extraDescriptionLabel.Text = message;
-        exceptionLabel.Text = exception;
-        errorDialog.PopupCenteredShrink();
-
-        onDialogDismissReturnToMenu = returnToMenu;
-        onDialogCloseCallback = onClosed;
+        errorDialog.ShowError(title, message, exception, returnToMenu, onClosed);
```

Comment:
This'll have a merge conflict with the save disallowed when extinct PR I opened today (there needs to be a feature to hide the copy exception button in the error dialog).

Reply:
Fixed the merge conflict.